test/Elastic.Apm.AspNetCore.Tests/Helper.cs

```diff
@@ -144,14 +144,16 @@ internal static void ConfigureServices(IServiceCollection services)
 			Startup.ConfigureServicesExceptMvc(services);
 			services
 #if NETCOREAPP3_0
-				.AddRazorPages()
+				.AddMvc()
 				.SetCompatibilityVersion(CompatibilityVersion.Version_3_0)
 #elif NETCOREAPP2_2
 				.AddMvc()
 				.SetCompatibilityVersion(CompatibilityVersion.Version_2_2)
-#else
+#elif NETCOREAPP2_1
 				.AddMvc()
 				.SetCompatibilityVersion(CompatibilityVersion.Version_2_1)
+#else
+				.AddMvc()
```

Comment:
Does it make sense to set the compatibility version to `3_1`?

Reply:
I think there is no `CompatibilityVersion.Version_3_1`. The doc page also does not exist for 3.1: https://docs.microsoft.com/en-us/aspnet/core/mvc/compatibility-version?view=aspnetcore-3.1 - if you switch to [3.0](https://docs.microsoft.com/en-us/aspnet/core/mvc/compatibility-version?view=aspnetcore-3.0) it already says:

> The SetCompatibilityVersion method is a no-op for ASP.NET Core 3.0 apps. That is, calling SetCompatibilityVersion with any value of CompatibilityVersion has no impact on the application.